core/gql/gql_mutations/mutation_by_filter.py

```diff
@@ -107,3 +107,46 @@ def _build_filters_from_gql_filters(filter_fields):
                 query_filter = F"{field}__{compare_type.lower()}"
                 fields.append(query_filter)
     return fields
+
+
+def mutation_on_uuids_from_filter_fs(django_object: django.db.models.Model,
+                                  object_gql_type: DjangoObjectType,
+                                  query_filters_field: str = 'extended_filters',
+                                  explicit_filters_handlers: Dict[str, str] = None,
+                                  return_objects: bool = False):
+    """
+    dedicated extended mutaion from flter decorator dedicated for Formal Sector entities. See doc string for
+    mutation_on_uuids_from_filter to read more how this works.
```

Comment:
please mention GistoryModel BusinessHistoryModel instead for FS (I hope that those model will be used in the future by other)

Reply:
ok, I can rename also this decorator to not use "fs". I will rename to "history_model"